src/Artemis.UI/Controllers/RemoteController.cs

```diff
@@ -24,17 +24,16 @@ public RemoteController(ICoreService coreService, IMainWindowService mainWindowS
         _router = router;
     }
 
-    [Route(HttpVerbs.Any, "/status")]
-    public void GetStatus()
+    [ResourceMethod(RequestMethod.Post, "bring-to-foreground")]
+    public void BringToForeground(IRequest request)
```

Comment:
I think your logic could be simplified with a `[FromBody] string? route` argument.

Reply:
Hey, I did try this, but it seems routes may not be omitted even if declared nullable?

```json
{
    "status": 400,
    "message": "Argument 'route' is expected to be read from the request body but the request does not contain any payload"
}
```

Reply:
Hm, I would consider this a bug. Sorry, will fix this in the next version.